config/webpack/demo/webpack.config.dev.js

```diff
@@ -0,0 +1,62 @@
+"use strict";
+
+var path = require("path");
+var glob = require("glob");
+var LodashModuleReplacementPlugin = require("lodash-webpack-plugin");
+
+var ROOT = process.cwd();
+var SRC = path.resolve("packages");
+var PACKAGES = glob.sync("packages/*/src", { root: ROOT });
+var FILES = PACKAGES.map(function(p) { return path.join(ROOT, p); });
+var DEMO = path.resolve("demo");
+var WDS_PORT = 3000;
+
+module.exports = {
+
+  devServer: {
+    port: parseInt(process.env.npm_package_config_wds_port_dev || WDS_PORT),
+    contentBase: "./demo",
+    noInfo: false
+  },
+
+  output: {
+    path: "./demo",
+    pathinfo: true,
+    filename: "main.js",
+    publicPath: "/assets/"
+  },
+
+  cache: true,
+  devtool: "source-map",
+  entry: {
+    app: ["./demo/app"]
+  },
+  stats: {
+    colors: true,
+    reasons: true
+  },
+  module: {
+    rules: [
+      {
+        // Transform source
+        test: /\.js$/,
+        // Use include specifically of our sources.
+        // Do _not_ use an `exclude` here.
+        include: FILES.concat([DEMO]),
+        // **Note**: Cannot use shorthand `"babel-loader"` or `"babel"` when
```

Comment:
NIT: You can remove this comment because modern webpack always requires full package name now for loaders.